packages/shared/src/components/MainFeedLayout.tsx

```diff
@@ -255,6 +263,41 @@ export default function MainFeedLayout({
 
   const disableTopPadding = (isFinder && isV1Search) || shouldUseFeedLayoutV1;
 
+  const tagsCount = feedSettings?.includeTags?.length || 0;
+  const isFeedPreviewEnabled = tagsCount >= REQUIRED_TAGS_THRESHOLD;
+  const [isPreviewFeedVisible, setPreviewFeedVisible] = useState(false);
+
+  if (
+    isOnboardingV4dot5 &&
+    alerts?.filter &&
+    feedName === SharedFeedPage.MyFeed
+  ) {
+    return (
+      <FeedPage className="relative !p-0">
```

Comment:
I'd personally avoid having 2 different renders.
Can't we simply make some props dynamic and only render the header dynamic?

Love to hear @capJavert thoughts as well :)

Reply:
I would agree yes, mostly because it would take two places to update Feed props for any change.

Reply:
Changed it to only use one render

Reply:
Looks good to me 👍

Reply:
@rebelchris thoughts about the change?

Reply:
Yeah it's good for me :)